vm/vm-runtime/src/force_upgrade_management.rs

```diff
@@ -9,7 +9,7 @@ pub const FORCE_UPGRADE_BLOCK_NUMBER: u64 = 17500000;
 
 pub fn get_force_upgrade_block_number(chain_id: &ChainId) -> u64 {
     if chain_id.is_test() {
-        50
+        10
```

Comment:
Please dont change this height smaller. It could cause some test cases failed.

Reply:
ok